pkg/nimble/rpble/include/nimble_rpble_params.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2019 Freie Universität Berlin
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup     net_nimble_netif_rpble
```

Comment:
This group still does not exist :wink:

Reply:
Yap, just noticed that I overlooked it :-)

Reply:
fixed